### DISCUSSION TITLE
how to add icon to the title bar?

### DISCUSSION THREAD

**Discussion by allentown521** [fuma-nama/fumadocs#1169] [Q&A]
![image](https://github.com/user-attachments/assets/a5d93893-7117-4ce2-b48d-8ceecfbce399)
like this

**Comment by nktnet1** [comment]
You can add it to `nav.title` in your base layout props. The `title` takes in a `ReactNode`:

- https://www.fumadocs.dev/docs/ui/layouts/nav#transparent-mode

Example [here](https://github.com/nktnet1/webview-kiosk/blob/dc4b51f81eab657e1632aae724fa4c4673234882/docs/src/lib/layout.shared.tsx#L10-L20).
